app/views/cloud_volume/edit.html.haml

```diff
@@ -1,18 +1,9 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "cloudVolumeFormController"}
+%form#form_div{:name => "angularForm", 'ng-controller' => "cloudVolumeFormController", "ng-cloak" => ""}
   = render :partial => "layouts/flash_msg"
   %h3
     = _('Edit Volume')
   .form-horizontal
-    .form-group
-      %label.col-md-2.control-label
-        = _('Volume Name')
-      .col-md-8
-        %input.form-control{:type          => "text",
-                            :name          => "name",
-                            'ng-model'     => "cloudVolumeModel.name",
-                            'ng-maxlength' => 128,
-                            :miqrequired   => false,
-                            :checkchange   => true}
+    = render :partial => "common_new_edit"
 
   %table{:width => '100%'}
     %tr
```

Comment:
@AparnaKarve I noticed there are some differences between the way how the button table is created for "new volume" than in the about partial layout. For example, in cloud volume case it checks if `@volume.id.nil?` to decide whether to show `Add` or `Save`, while the layout partial uses `newRecord`.

I guess we should update the whole controller/view to make it compliant with the partial layout. Perhaps in a separate PR. What do you think?